### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/get-started/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#506] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/get-started/intro.html

主题初衷 制作本主题的初衷是发现 VuePress 默认的主题只是一个提供基础文档布局的主题。 例如，它不会为 SEO 优化注入元标记，也不会生成 Sitemap 来帮助搜索引擎索引文档的内容。 VuePress 虽然在一定程度上扩展了 Markdown 语法，但仍然缺少一些常用的功能，如文本对齐、标记、流程图、公式、展示等，同时默认主题提供的一些功能...

https://theme-hope.vuejs.press/zh/guide/get-started/intro.html

**Comment by sciencekiller** [comment]
感谢Mr.Hope

**Comment by shb-mooncake** [comment]
1111
